src/com/layout/row/row.js

```diff
@@ -0,0 +1,15 @@
+import {h, Component} 				from 'preact/preact';
+
+export default class Row extends Component {
+	constructor( props ) {
+		super(props);
+	}
+
+	render( props ) {
+		return (
+			<div {...props} class="-row">
+        {props.children}
+      </div>
```

Comment:
Indentation looks weird

Reply:
The indentation issues are weird, not indented like that on my local copy but pushed like that... I'll see if I can fix it.